templates/payment-form.html

```diff
@@ -22,16 +22,57 @@ <h1>
   <div class="column">
```

Comment:
Copy change requests:

[Line 16](https://github.com/Techtonica/techtonica.org/pull/346/files#diff-c57d661d30fd9c779c2e76990d63db2d9794dc3dbf0bfc8f0bf68f7e8011d95eR16): `Post A Job`

[Line 19](https://github.com/Techtonica/techtonica.org/pull/346/files#diff-c57d661d30fd9c779c2e76990d63db2d9794dc3dbf0bfc8f0bf68f7e8011d95eR19): `Have a job opening but your company isn’t ready to become a hiring partner?
You can support Techtonica today by making a $100 donation and we will share your job with our past graduates. Our 100+ graduates have 0-5 years of experience in all areas of full stack development.`

Reply:
Addressed in [2c5745b](https://github.com/Techtonica/techtonica.org/pull/346/commits/2c5745b9a1d121f0d16e42983fed9b20450329c1)